### DISCUSSION TITLE
DataTable set column to fill

### DISCUSSION THREAD

**Discussion by gaborbernat** [Textualize/textual#5092] [Q&A]
How do I set a column in a data table to take up as much space as left after filling the other columns? Think of a description column that can be long. Ideally, the table would stretch over multiple rows, but if that is not possible to be cut off. I would rather not set a static limit because if the terminal fits in width it should fit as much as possible.

**Comment by willmcgugan** [comment]
That's not supported at the moment. Currently columns can be fixed with, or optimal width.
